Lection2/task2.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import "fmt"
+
+type Point struct {
+	x, y int
+}
+
+type Square struct {
+	start Point
+	a     uint
+}
+
+// We don't want to change start point. Are we?
```

Comment:
This is not an informative comment. Please remove it. Could you also add an informative one? Just to describe this method.